src/main/java/com/joss/conductor/mobile/Locomotive.java

```diff
@@ -314,8 +310,12 @@ public boolean isPresentWait(String id, long timeOutInSeconds) {
     }
 
     public boolean isPresentWait(By by, long timeOutInSeconds) {
```

Comment:
Perhaps increase the flexibility of this method by allowing for chaining of selectors. Not sure of the exact syntax but something like this:
```
public boolean isPresentWait(By... selectors, long timeOutInSeconds) {
  try {
       for (By selectors : selector) {
            waitForCondition(visibilityOfAllElementsLocatedBy(selector), timeOutInSeconds, 500);
       }
   } catch (TimeoutException e) {
        return false;
   }
}
```

Reply:
I don't think I'll have time to test these changes out. Would you mind making a feature request?
(Just being a little paranoid with how late we figured out the actual behavior of `isPresentWait` had changed)

Reply:
I don't think chaining is needed. If wanted then feature request would be nice.

If there is a good use-case for chaining, and will increase your QOL then a feature request would be awesome

Reply:
It is more about convenience than need and increased flexibility. Just eliminates the need to re-type `getDriver().isPresentWait()` over and over. I'm not super passionate about it, especially since I no longer am a consumer of this conductor-mobile so it was just a "while you are already in there" type idea.